packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenPage/AccessTokenPage.stories.ts

```diff
@@ -1,17 +1,35 @@
 import { Meta, StoryObj } from '@storybook/react'
 import { AccessTokenPage } from '../AccessTokenPage'
+import { MOCK_REPO_ORIGIN } from '../../../utils/functions/getEndpoint'
+import { getPersonalAccessTokenHandlers } from '../../../mocks/msw/handlers/personalAccessTokenHandlers'
 
 const meta = {
   title: 'Synapse/AccessTokenPage',
   component: AccessTokenPage,
+  args: {
+    title: 'Personal Access Tokens',
+    body: 'Create and manage tokens that can be used to access your Synapse account programmatically.',
+  },
+  parameters: { stack: 'mock' },
```

Comment:
By default use the mock stack. I added a few handlers so now you don't have to log in to see these stories